packages/target-decisioning-engine/src/artifactProvider.js

```diff
@@ -110,13 +112,12 @@ async function ArtifactProvider(config) {
         let responseData;
         if (res.ok && res.status === OK) {
           responseData = await res.json();
-
           const etag = res.headers.get("Etag");
           if (etag != null && isDefined(etag)) {
             lastResponseData = responseData;
             lastResponseEtag = etag;
           }
-
+          geoContext = createGeoObject(res);
```

Comment:
Shouldn't you also be calling eventEmitter(GEO_LOCATION_UPDATED, geoContext) somewhere?  Did I miss it? Seems like it should be in emitNewArtifact() right after the ARTIFACT_DOWNLOAD_SUCCEEDED call

Reply:
Yeah I'm calling it in `geoProvider` https://github.com/jasonwaters/target-nodejs-sdk/pull/16/files#diff-a746d04bc421a9b5fa1be5b92ba6dd7dR84

Reply:
@mptap How does the code in geoProvider get called when a new artifact is downloaded in artifactProvider?  🤔 The `GEO_LOCATION_UPDATED` event needs to be emitted when geo is extracted from the artifact headers as well as when geoProvider makes an explicit akamai call to `/v1/geo` for just the geo.

Reply:
@jasonwaters I see what you mean. I was earlier thinking of validGeoRequestContext. I've now added `GEO_LOCATION_UPDATED ` in emitNewArtifact() upon ARTIFACT_DOWNLOAD_SUCCEEDED firing.